bin/plugin/lib/git.js

```diff
@@ -47,7 +47,7 @@ async function commit( gitWorkingDirectoryPath, message, filesToAdd = [] ) {
  * Creates a local branch.
  *
  * @param {string} gitWorkingDirectoryPath Local repository path.
- * @param {string} branchName Branch Name
+ * @param {string} branchName              Branch Name
```

Comment:
Not a blocker, but we should look at standardizing either including or excluding the `.` period at the end of the @param description, PHP Coding Standards require this, so I'd lean toward that also for JS (A goal is to align the CS between languages as best we can when suited)